### PR TITLE
fix(aio): style mods and content udpates

### LINKED ISSUE #16094: Stale news headline on the front page of aio
**I'm submitting a ...**  (check one with "x")
`[x]` bug report

**Current behavior**
The main headline on the front page is stale referring to "Join us for DEVintersection Amsterdam Nov 14-16!REGISTER NOW".

-----
screen shot 2017-04-18 at 5 54 27 pm

----

**Expected behavior**
We should copy the headline from the current angular.io and refer to Angular v4.0 instead.

### LINKED ISSUE #16122: Content errors in the the main docs page
**Current behavior**
Navigate to https://aio-staging.firebaseapp.com/docs and you'll see:

screen shot 2017-04-18 at 8 01 53 pm

1. The page should not claim that version of the docs is 4.0.0 and there should be no space between "v" and "4". We should say that these are "v4" docs unless we rea
lly want to update them every time we cut a new minor.
2. Mentioning the changelog without a link is not useful. I don't know what's the purpose of the changelog for the docs that is separate from the changelog of Angular itself. We should find a way to unite these two an in the meantime not mention it here at all.
3. There is a typo in the last sentence about v2 docs.
4. The developer guide link is broken, it leads to `https://aio-staging.firebaseapp.com/..api`.

### LINKED ISSUE #16124: Quickstarts are missing headers
**Current behavior**

Basic quickstart:
screen shot 2017-04-18 at 8 34 49 pm

-----
CLI quickstart:
screen shot 2017-04-18 at 8 34 56 pm

### PR DESCRIPTION
**Pull request by sjtrimble**
Fixed capitalization in sidenav menu items
Add min height property to sidenav-content for better viewing on short pages or high resolution screens
Add dividier between change log items as before to h2
Table adjustments to reduce off-screen items
Content updates including capitalization consistency and title additions

Closes https://github.com/angular/angular/issues/16124
Closes https://github.com/angular/angular/issues/16094
Closes https://github.com/angular/angular/issues/16122

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[ ] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: aio/content/guide/browser-support.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Browser support and polyfills guide.
 Angular supports most recent browsers. This includes the following specific versions:
 
 
-<table>
+<table class="fixed-table">
```
Comment (gkalpak) [inline aio/content/guide/browser-support.md]: _fixed-table_ --> _table-fixed_

File: aio/src/styles/1-layouts/_content-layout.scss [hunk 2]
```diff
@@ -20,4 +21,12 @@
 
 .sidenav-content button {
   min-width: 50px;
+}
+
+#guide-change-log h2::before {
```
Comment (gkalpak) [inline aio/src/styles/1-layouts/_content-layout.scss]: Such selectors (that rely on auto-generated stuff) are brittle. Could we somehow target the specific `h2` explicitly (e.g. by adding a class/id to it)?

File: aio/src/styles/2-modules/_table.scss [hunk 3]
```diff
@@ -4,6 +4,10 @@ table {
   border-radius: 2px;
   background: $offwhite;
 
+  &.table-fixed {
+    table-layout: fixed;
```
Comment (gkalpak) [inline aio/src/styles/2-modules/_table.scss]: Why this?

**Review by gkalpak — CHANGES_REQUESTED** [review]

File: aio/src/styles/1-layouts/_content-layout.scss (hunk 2 above)
Comment (sjtrimble) [inline aio/src/styles/1-layouts/_content-layout.scss]: Not sure what the easiest way would be. The intent was to have each change log section separated by an hr. We could do this by changing the authored document by either adding the hr itself, or by moving the sections into div's labeled with appropriate classes, and then I can easily add an after that way.

Do you have a recommendation on which one would be more appropriate. I'd prefer the authoring changes with divs for sections + classes.

Comment (gkalpak) [inline aio/src/styles/1-layouts/_content-layout.scss]: Actually, after realizing this targets all `h2` element inside that document and not a specific one, I think thisis fine as it is.

**Review by gkalpak — APPROVED** [review]

**Pull request closed without merging**